### PR TITLE
Remove arguments of ecma_new_values_collection.

### PR DESCRIPTION
**Pull request by zherczeg**

### CODE REVIEW

**Comment by zherczeg** [comment]
Should be landed before #2196.

File: jerry-core/ecma/base/ecma-globals.h [hunk 1]
```diff
@@ -1052,6 +1052,14 @@ typedef double ecma_number_t;
  * value as well.
  */
 
+/**
+ * JerryScript init flags.
```
Comment (galpeter) [inline jerry-core/ecma/base/ecma-globals.h]: Incorrect enum description.

**Comment by zherczeg** [comment]
Thx, fixed.

**Comment by yichoi** [comment]
LGTM

**Pull request merged**